gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -61,6 +61,7 @@ public abstract class LaunchConfigTask extends DefaultTask {
     private static final ImmutableList<String> java15Options =
             ImmutableList.of("-XX:+UnlockDiagnosticVMOptions", "-XX:+ExpandSubTypeCheckAtParseTime");
     private static final ImmutableList<String> disableBiasedLocking = ImmutableList.of("-XX:-UseBiasedLocking");
+    private static final ImmutableList<String> disableC2 = ImmutableList.of("-XX:TieredStopAtLevel=1");
```

Comment:
hang on guys - if you're just trying to turn off the C2 compiler, shouldn't this say `-XX:TieredStopAtLevel=3`

https://www.baeldung.com/jvm-tiered-compilation#4-level-3--full-c1-compiledcode

Reply:
Try some benchmarks ;-)

Reply:
I’ve forwarded some info internally

Reply:
Fascinating! Is it worth calling out the expected performance impact here though?? If an application was CPU bound then disabling C2 could make it run ~10x slower than before? Although maybe IO bound stuff wouldn't see as much of a slowdown?

Reply:
For any future readers - we found a better way of doing this:

```
-XX:CompileCommand=exclude,sun/security/ssl/SSLEngineInputRecord.decodeInputRecord
```

https://github.com/palantir/sls-packaging/pull/1372